composer.json

```
@@ -10,7 +10,7 @@
     ],
     "require": {
         "php": "^7.1 || ^8.0",
```

As discussed earlier.

```suggestion
        "php": "^7.2 || ^8.0",
```